docs/manual/src/docs/asciidoc/_includes/servlet/saml2/saml2-login.adoc

```diff
@@ -1610,3 +1610,33 @@ http {
 The success handler will send logout requests to the asserting party.
 
 The request matcher will detect logout requests from the asserting party.
+
+[[servlet-saml2login-store-authn-request]]
+=== Storing the Authentication Request
+
+The `Saml2AuthenticationRequestRepository` is responsible for the persistence of the `AbstractSaml2AuthenticationRequest` from the time the Authentication Request is initiated to the time the Authentication Response is received.
```

Comment:
You might consider linking back to `[[servlet-saml2login-sp-initiated-factory]]` and `[[servlet-saml2login-authenticate-responses]]` to help folks back to the sections that give an overview of how Spring Security handles AuthnRequests and SAML Responses.